composer.json

```diff
@@ -14,13 +14,14 @@
         "issues": "https://github.com/liip/metadata-parser/issues"
     },
     "require": {
-        "php": "^7.2 || ^8.0",
+        "php": "^7.4 || ^8.0",
         "ext-json": "*",
+        "doctrine/annotations": "^1.13 || ^2.0.1",
         "psr/log": "^1|^2|^3"
     },
     "require-dev": {
         "doctrine/collections": "^1.6",
-        "friendsofphp/php-cs-fixer": "v2.18.7",
+        "friendsofphp/php-cs-fixer": "v3.17.0",
```

Comment:
looking at the [lowest version build](https://github.com/liip/metadata-parser/actions/runs/5134933356/jobs/9239640381?pr=36) we still end up with doctrine/annotations 2, and i tracked it down to the cs fixer. let me try something

Reply:
https://github.com/liip/metadata-parser/pull/37